importer/process_ios_assets.py

```diff
@@ -89,8 +89,7 @@ def process_assets():
                     "author": "xcode"
                 },
                 "properties": {
-                    "template-rendering-intent": rendering_intent,
-                    "preserves-vector-representation": True
```

Comment:
Don't we wanna keep the property, but set it to False?

Reply:
The default value of the "preserves-vector-representation" is false so not necessary to make it explicit.